models/list_option/list_option.go

```diff
@@ -46,7 +47,7 @@ func ListOptionAll() []*ListOptionStruct {
 	return slice
 }
 
-func FindByAction(action string) *ListOptionStruct {
+func FindByAction(action string) (*ListOptionStruct, error) {
```

Comment:
__[golint]__
* [golint] exported function FindByAction should have comment or be unexported